js/ui/collection/ui.collection_widget.base.js

```diff
@@ -105,6 +120,13 @@ var CollectionWidget = Widget.inherit({
             */
             selectOnFocus: false,
 
+            /**
+            * @name CollectionWidgetOptions.repaintChangesOnly
+            * @type boolean
+            * @hidden
+            */
+            repaintChangesOnly: false,
```

Comment:
Useless option or unclear name. If you can repaint only the changed part of the widget, in what cases the customer should disable this functionality?

Otherwise, when you could not repaint only the part of the widget, the name of the option is not clear. What does this option do?

//cc @RomanTsukanov , @viterobk